packages/lesswrong/server/search/elastic/ElasticClient.ts

```diff
@@ -64,6 +70,28 @@ class ElasticClient {
     }
     return this.client.search(request);
   }
+
+  async multiSearch(queryData: MultiQueryData): Promise<ElasticSearchResponse> {
+    const query = new ElasticMultiQuery(queryData);
+    const request = query.compile();
+    if (DEBUG_LOG_ELASTIC_QUERIES) {
+      // eslint-disable-next-line no-console
+      console.log("Elastic multi query:", JSON.stringify(request, null, 2));
+    }
+    const result: ElasticSearchResponse = await this.client.search(request);
+
+    const aggregation = result.aggregations?.indexes as undefined | {
+      buckets?: {hits?: AggregationsTopHitsAggregate}[],
+    };
+    const allHits = (aggregation?.buckets ?? [])
+      .flatMap((bucket) => bucket?.hits?.hits?.hits)
+      .filter((hit): hit is SearchHit<SearchDocument> => (hit?._score ?? 0) > 0);
+    const sortedHits = sortBy(allHits ?? [], ((hit) => hit._score));
+    result.hits.hits = sortedHits;
+    result.hits.max_score = sortedHits[0]?._score ?? null;
+    delete result.aggregations;
```

Comment:
curious why do we need to do this

Reply:
Technically we don't _need_ to, but it means that the return value of `multisearch` is the same shape as the return value of `search`, and we don't need the aggregations anymore since we already used them for the relevancy ordering above.